riotfile.py

```diff
@@ -1837,7 +1837,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="rq",
-            command="pytest tests/contrib/rq",
+            command="pytest {cmdargs} tests/contrib/rq",
```

Comment:
Why'd we need to add this?

Reply:
I think I figured it out in https://github.com/DataDog/dd-trace-py/pull/6772#issuecomment-1696498213